assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -285,14 +293,23 @@ const SelectedVehicleDataLayers = ({
             </>
           )}
           {showShapeAndStops && routePatternForVehicle && (
-            <RoutePatternLayers
-              routePattern={routePatternForVehicle}
-              isSelected={false}
-            />
+            <>
+              <RoutePatternLayers
+                routePattern={routePatternForVehicle}
+                isSelected={false}
+              />
+            </>
```

Comment:
**nitpick:** since there's only one element here I think you can get rid of the enclosing `<>`...`</>`.